modyn/selector/internal/selector_strategies/downsampling_strategies/rho_loss_downsampling_strategy.py

```diff
@@ -39,20 +39,37 @@ def __init__(
         rho_pipeline_id, data_config = self._get_or_create_rho_pipeline_id_and_get_data_config()
         self.rho_pipeline_id: int = rho_pipeline_id
         self.data_config = data_config
-        self.il_model_id: Optional[int] = None
 
     def inform_next_trigger(self, next_trigger_id: int, selector_storage_backend: AbstractStorageBackend) -> None:
         if not isinstance(selector_storage_backend, DatabaseStorageBackend):
             raise ValueError("RHOLossDownsamplingStrategy requires a DatabaseStorageBackend")
 
         self._prepare_holdout_set(next_trigger_id, selector_storage_backend)
-        self.il_model_id = self._train_il_model(next_trigger_id)
+        self._train_il_model(next_trigger_id)
```

Comment:
we couldn't simply store the `il_model_id` for use later, because there are multiple selector instances per pipeline, the request to fetch downsampling config can be routed to another unsynced downsampler.

So we have to fetch data from database.

Reply:
Good catch!